tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -146,58 +224,30 @@ def test_createami_wrong_pcluster_version(
     # Therefore, we can bypass the version check in CLI and test version check of .bootstrapped file in Cookbook.
     wrong_ami = pcluster_ami_without_standard_naming(wrong_version)
 
-    command = _compose_command(region, instance, os, request, vpc_stack, wrong_ami, cluster_config)
-    _createami_and_assert_error(command, fr"AMI was created.+{wrong_version}.+is.+used.+{current_version}")
-
-
-def _compose_command(region, instance, os, request, vpc_stack, base_ami, cluster_config):
-    # Networking
-    vpc_id = vpc_stack.cfn_outputs["VpcId"]
-    networking_args = ["--vpc-id", vpc_id, "--subnet-id", vpc_stack.cfn_outputs["PublicSubnetId"]]
-
-    # Custom Cookbook
-    custom_cookbook = request.config.getoption("createami_custom_chef_cookbook")
-    custom_cookbook_args = [] if not custom_cookbook else ["-cc", custom_cookbook]
-
-    return (
-        [
-            "pcluster",
-            "createami",
-            "-ai",
-            base_ami,
-            "-os",
-            os,
-            "-r",
-            region,
-            "-c",
-            cluster_config.as_posix(),
-            "-i",
-            instance,
-        ]
-        + custom_cookbook_args
-        + networking_args
+    cluster_config = pcluster_config_reader(
+        config_file="image.config.yaml",
+        parent_image=wrong_ami,
+        instance_type=instance,
     )
+    image_id = "-".join(["integ", "test", "build", "image", "wrong", "version", generate_random_string()])
+
+    result = build_image(image_id, region, cluster_config)
+
+    _assert_build_image_failed(result, image_id, region)
+    # TODO get cloudwatch log
```

Comment:
I have a task for it. I'm creating the commands to retrieve the logs: https://github.com/aws/aws-parallelcluster/pull/2863 

We'll add them in the factory to be used when needed.